### PR TITLE
III-4922 - rounded corners

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

-  Rounded corners

---

Ticket: https://jira.uitdatabank.be/browse/III-4922

Related [udb3-angular-app PR](https://github.com/cultuurnet/udb3-angular-app/pull/180)

### CODE REVIEW

**Comment by brampauwelyn** [comment]
No rounded corners on the image & video upload box?

Screenshot 2022-10-10 at 15 56 01

+ not much contrast between background of box & the button

File: src/ui/DatePicker.tsx [hunk 1]
```diff
@@ -43,6 +43,18 @@ const DatePicker = ({
       minDate={minDate}
       maxDate={maxDate}
       customInput={<Input id={id} />}
+      css={`
+        .react-datepicker {
+          border-radius: 8px;
```
Comment (brampauwelyn) [inline src/ui/DatePicker.tsx:48]: I wonder if we should put this border-radius value in some kind of global variable?
I see 8px is being used in a lot of places? 
Maybe nice if we want to change the value later?

File: src/ui/Input.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ const Input = forwardRef(
       placeholder={placeholder}
       className={className}
       maxWidth="43rem"
-      css="border-radius: 0;"
+      css="border-radius: 8px;"
```
Comment (brampauwelyn) [inline src/ui/Input.tsx:77]: use global variable?

File: src/ui/Modal/ContentModal.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ const ContentModal = ({
       }
 
       .modal-content {
-        border-radius: 0;
+        border-radius: 8px;
```
Comment (brampauwelyn) [inline src/ui/Modal/ContentModal.js:39]: use global variable?

File: src/ui/Pagination.tsx [hunk 4]
```diff
@@ -65,7 +65,9 @@ const Pagination = ({
           border-color: ${getValue('borderColor')};
           padding: ${getValue('paddingY')} ${getValue('paddingX')};
 
-          &:hover {
+          border-radius: 8px;
+
+          */ &:hover {
```
Comment (brampauwelyn) [inline src/ui/Pagination.tsx:70]: Should this be part be in comment? 
Or can it be removed?

```suggestion
           &:hover {
```

**Review by brampauwelyn — CHANGES_REQUESTED** [review]
Looks good! Love the rounded 🌽ers

**Comment by simon-debruijn** [comment]
TODO:

Fix active state

Screenshot 2022-10-11 at 17 07 07

**Comment by simon-debruijn** [comment]
@brampauwelyn 

Screenshot 2022-10-12 at 10 31 15

**Review by brampauwelyn — APPROVED** [review]

**Pull request merged**